lib/rubocop/cop/rails/short_i18n.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop enforces that short forms of `I18n` methods are used:
+      # `t` instead of `translate` and `l` instead of `localize`.
+      #
+      # This cop has two different enforcement modes. When the EnforcedStyle
+      # is conservative (the default) then only `I18n.translate` and `I18n.localize`
+      # calls are added as offenses.
+      #
+      # When the EnforcedStyle is aggressive then all `translate` and `localize` calls
+      # without a receiver are added as offenses.
+      #
+      # @example EnforcedStyle: conservative (default)
+      #   # bad
+      #   I18n.translate :key
+      #   I18n.localize Time.now
+      #
+      #   # good
+      #   I18n.t :key
+      #   I18n.l Time.now
+      #
+      # @example EnforcedStyle: aggressive
+      #   # bad
+      #   I18n.translate :key
+      #   translate :key
+      #   I18n.localize Time.now
+      #   localize Time.now
+      #
+      #   # good
+      #   I18n.t :key
+      #   t :key
+      #   I18n.l Time.now
+      #   l Time.now
```

Comment:
Can you update to the following example because it would like to highlight the differences between the options?

```suggestion
      # @example
      #   # bad
      #   I18n.translate :key
      #   I18n.localize Time.now
      #
      #   # good
      #   I18n.t :key
      #   I18n.l Time.now
      #
      # @example EnforcedStyle: conservative (default)
      #   # good
      #   translate :key
      #   localize Time.now
      #   t :key
      #   l Time.now
      #
      # @example EnforcedStyle: aggressive
      #   # bad
      #   translate :key
      #   localize Time.now
      #
      #   # good
      #   t :key
      #   l Time.now
```